app/src/backend/formulas.ts

```diff
@@ -372,13 +379,14 @@ export function handleMethaneCommitmentFormula(
     METHANE_FRACTION *
     (16 / 12.0);
 
-  const ch4Emissions =
-    totalSolidWaste *
+  const ch4Emissions = Decimal.mul(
+    totalSolidWaste,
     methaneGenerationPotential *
-    (1 - recoveredMethaneFraction) *
-    (1 - oxidationFactor);
+      (1 - recoveredMethaneFraction) *
+      (1 - oxidationFactor),
```

Comment:
```suggestion
      1 - recoveredMethaneFraction,
      1 - oxidationFactor,
```
To prevent number overflows